src/assets/scripts/client/speech.js

```diff
@@ -21,12 +21,39 @@ export const speech_init = () => {
     }
 };
 
+export const randomize_pilot_voice = () => {
```

Comment:
should be camel-case without underscores. names with underscores are legacy and should be replaced in the future.

Reply:
Also, the function needs a docblock

Reply:
I was thinking about keeping the whole file with the same format and refactor whole underscore case when the time comes. Should both formats be mixed until we change all of them to camelcase?

Reply:
Ordinarily yes, however, my vote here is to leave it for now simply because I have an issue out there to refactor this entire module.